.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+elso please read our [Structure Convention](https://github.com/hey-car/hey-ui/blob/main/STRUCTURE_CONVENTION.md) and [Style Guide](https://github.com/hey-car/hey-ui/blob/main/STYLEGUIDE_CONVENTION.).
+Please read the full text so that you can understand what actions will and will not be tolerated.
+
+## Open Development
+
+All work on Hey-UI happens directly on [GitHub](https://github.com/hey-car/hey-ui). Both core team members and external contributors send pull requests which go through the same review process.
+
+## Branch Organization
+
+According to our [release schedule](changelog#Release-Schedule), we maintain two branches, `master` and `feature`. If you send a bugfix pull request, please do it against the `master` branch, if it's a feature pull request, please do it against the `feature` branch.
+
+`main` - Contains only tested code. Сode gets here from `release/*` and `hotfix/*` branches.
+
+`release/*` - Contains branches formed from the `develop` branch, for future release, e.g. `release/0.0.1`.
+
+`develop` - General branch for developing. Сode gets here from `feature/*` and `bugfix/*` branches.
+
+`feature/*` - Branches are created based on ticket name, e.g. `feature/HEYUI-999-status-button-component`. Every individual ticket must have its corresponding branch, even for small tasks.
+
+`hotfix/*` - These are branches that contains very urgent fixes for production, e.g. `hotfix/HEYUI-999-fix-button-attribute`.
+
+`bugfix/*` - These branches contain bug fixes and can be merged into `develop`, `release` or `feature` braches, e.g. `bugfix/HEYUI-999-fix-button-attribute`.
+
+## Bugs
+
+We are using [GitHub Issues](https://github.com/hey-car/hey-ui/issues) for bug tracking.
+
+Before you report a bug, please make sure you've searched existing issues, and read our [FAQ](/docs/react/faq).
+
+## Your First Pull Request
+
+Working on your first Pull Request? You can learn how from this free video series:
+
+[How to Contribute to an Open Source Project on GitHub](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github)
+
+If you decide to fix an issue, please be sure to check the comment thread in case somebody is already working on a fix. If nobody is working on it at the moment, please leave a comment stating that you intend to work on it so other people don't accidentally duplicate your effort.
+
+If somebody claims an issue but doesn't follow up for more than two weeks, it's fine to take over it but you should still leave a comment.
+
+## Sending a Pull Request
+
+The core team is monitoring for pull requests. We will review your pull request and either merge it, request changes to it, or close it with an explanation.
+
+**Before submitting a pull request**, please make sure the following is done:
+
+1. Fork the repository and create your branch from the [correct branch](#Branch-Organization).
```

Comment:
there are only `1.`s in this list; please change it into an unordered list or order it accordingly